src/views/v2/emails/template-parts/body/order/purchaser-details.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * Event Tickets Emails: Order Purchaser Details
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/emails/template-parts/body/order/purchaser-details.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link https://evnt.is/tickets-emails-tpl Help article for Tickets Emails template files.
+ *
+ * @version TBD
+ *
+ * @since TBD
+ *
+ * @var Tribe_Template  $this  Current template object.
+ * @var Module           $provider              [Global] The tickets provider instance.
+ * @var string           $provider_id           [Global] The tickets provider class name.
+ * @var \WP_Post         $order                 [Global] The order object.
+ * @var int              $order_id              [Global] The order ID.
+ * @var bool             $is_tec_active         [Global] Whether `The Events Calendar` is active or not.
+ */
+
+// @todo @codingmusician @juanfra Replace hardcoded data with dynamic data.
+
+?>
+<tr>
+	<td>
+		<table>
```

Comment:
At this point, for templates in which we're having really basic layout that's supported by any email client, we should go with divs and basic HTML elements. If we end up using too many tables, the customization experience will be not ideal.

Reply:
We can double-check if things are supported [here](https://www.caniemail.com/), for example [the `<div>` tag](https://www.caniemail.com/features/html-div/).